quil-py/quil/program/__init__.pyi

```diff
@@ -1,3 +1,70 @@
+"""
+The `quil.program` module contains classes for constructing and representing a Quil program.
+
+# Examples
+
+## Source Mapping for Calibration Expansion
+
+```py
```

Comment:
Is this snippet run and tested?  It would be nice if it were but I don't know enough about Python to tell if that's happening/how easy it would be to have that happen.

Reply:
yes, it's a test case

Reply:
Can we add a pointer to the test case it corresponds to?